wrappers/javascript/indy-vdr-react-native/cpp/indyVdr.cpp

```diff
@@ -240,11 +240,13 @@ jsi::Value buildGetNymRequest(jsi::Runtime &rt, jsi::Object options) {
   auto seqNo = jsiToValue<int32_t>(rt, options, "seqNo", true);
```

Comment:
`jsiToValue` could take an optional param which sets the undefined value (-1) in this case. 

Not really required to change, but it can be a nice addition in the future if we run into this again.